src/content/manifest/overview-manifest.mdx

```diff
@@ -8,105 +8,109 @@ id: overview-manifest
-
-There are other sections to help you build your ideal development experience in Okteto, but this document focuses solely on introducing you to the core concepts of the Okteto manifest. You can learn more about the other Okteto manifest options in the [manifest reference](reference/okteto-manifest.mdx).
+There are other sections to help you build your ideal development experience in Okteto, but this document focuses solely on introducing you to the core concepts of the Okteto Manifest.
+You can learn more about the other Okteto Manifest options in the [manifest reference](reference/okteto-manifest.mdx).
 
 ## Build
 
-This section contains the instructions for Okteto to build the images for each service in your application. You can specify a specific subfolder for the image context and use other configuration options to customize how your images are built.
+This section contains the instructions for Okteto to build the images for each service of your application.
 
 Your `build` section might look like this:
 
 ```yaml
 build:
-  base:
-    context: .
   api:
     context: api
   frontend:
     context: frontend
     dockerfile: Dockerfile
-    target: dev
-    depends_on: base
-    args:
-      SOURCE_IMAGE: ${OKTETO_BUILD_BASE_IMAGE}
     secrets:
       npmrc: .npmrc
 ```
 
-This configuration defines images to be built for three services: `base`, `api`, and `frontend`. It also defines a `context` for each image, which tells Okteto which folder/subfolder to use for building each container image. For example: `default (.)` tells Okteto to use the current folder for the `base` image, the `api` subfolder for the `api` image, and the `frontend` subfolder for the `frontend` image.
+This configuration defines images to be built for three services: `api`, and `frontend`. It also defines a `context` for each image, which tells Okteto which folder/subfolder to use for building each container image.
+In this case, Okteto is using the `api` subfolder for the `api` image, and the `frontend` subfolder for the `frontend` image. For the `frontend` image, it's also defining the `dockerfile` path and a build secret.
 
-You can learn more about these additional parameters [here](reference/okteto-manifest.mdx#build-object-optional).
+You can learn more about other `build` parameters [here](reference/okteto-manifest.mdx#build-object-optional).
 
 ## Deploy
 
-This section tells Okteto how, specifically, to deploy your development environment. It typically uses a combination of `helm`, `kubectl`, and `okteto` commands. You can execute multiple commands and label them.
-
-All services defined in this section will appear in the Okteto UI as part of a single development environment that represents the whole of your application. Each individual service will appear as its own development container that’s part of the development environment for your application.
-
-:::note
```

Comment:
Is it worth retaining the note about external resources? Agree with removing the rest.